script/sync_prod_to_gcp/resource/bin/arxiv-mail

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+Sending email to non-local MTA. Almost drop in replacement of /usr/bin/mail
+
+This is intended to be on sync node's /us/local/bin/arxiv-mail.
+You can use this for any MTA but YMMV.
+
+"""
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+import sys
+import os
+
+
+def _send_email(smtp_server, smtp_port, smtp_user, smtp_pass, recipient, sender, subject):
+    sys.exit(0)
+
+def send_email(smtp_server, smtp_port, smtp_user, smtp_pass, recipient, sender, subject):
+    body = sys.stdin.read()
+
+    msg = MIMEMultipart()
+    msg['From'] = sender
+    msg['To'] = recipient
+    msg['Subject'] = subject
+
+    msg.attach(MIMEText(body, 'plain'))
+
+    try:
+        with smtplib.SMTP(smtp_server, smtp_port) as server:
+            server.ehlo()
```

Comment:
No need to change this but there must be a better library for sending email than this.